pyrefinebio/exceptions.py

```diff
@@ -1,38 +1,44 @@
-class ServerError(Exception):
+class Error(Exception):
+    def __init__(self, message=None):
+        self.message = message
+        super().__init__(self.message)
```

Comment:
What's the reason for this change and what's it doing?

Reply:
the goal was to expose the fully formatted message to be used like this:

```
try:
    something.something()
except pyrefinebio.exceptions.DownloadError as e:
    raise OtherException(e.message)
```

previously I would use `e.base_message` which would be weird unformatted stuff for some of the errors

I wasn't sure if this was the best way to do this, but it was the easiest for me for right now.
I'm thinking of refactoring all of these exceptions as a part of #26 anyway but we'll see.
if there is a better way lmk though!